code/modules/religion/religion_sects.dm

```diff
@@ -216,3 +216,32 @@
 	adjust_favor(20, user) //it's not a lot but hey there's a pacifist favor option at least
 	qdel(offering)
 	return TRUE
+
+/**** Carp Sect ****/
+
+/datum/religion_sect/carp_sect
+	name = "Followers of the Great Carp"
+	desc = "A sect dedicated to the space carp and carp'sie."
+	convert_opener = "Drown the station in fish and water.<br>Sacrificing meat grants you favor."
+	alignment = ALIGNMENT_NEUT
+	max_favor = 10000
+	desired_items = list(/obj/item/reagent_containers/food/snacks/meat)
+	rites_list = list(/datum/religion_rites/summon_carp, /datum/religion_rites/flood_area, /datum/religion_rites/summon_carpsuit)
+	altar_icon_state = "convertaltar-blue"
+
+//Carp bibles give people the carp faction!
+/datum/religion_sect/carp_sect/sect_bless(mob/living/L, mob/living/user)
+	if(isliving(L))
+		L.faction |= "carp"
+		user.visible_message("<span class='notice'>[user] blessed [L] with the power of [GLOB.deity]! They are now protected from Space Carps, Although carps will still fight back if attacked.</span>")
+		SEND_SIGNAL(L, COMSIG_ADD_MOOD_EVENT, "blessing", /datum/mood_event/blessing)
+	return TRUE
```

Comment:
```suggestion
	if(!isliving(L))
		return FALSE
	L.faction |= "carp"
	user.visible_message("<span class='notice'>[user] blessed [L] with the power of [GLOB.deity]! They are now protected from Space Carps, Although carps will still fight back if attacked.</span>")
	SEND_SIGNAL(L, COMSIG_ADD_MOOD_EVENT, "blessing", /datum/mood_event/blessing)
	return TRUE
```

Also unsure if you really want this being a visible message to everyone.

Reply:
Mainly kept it as a visible message so that people dont assume oh! antag! tbh